### PR TITLE
Added necessary information in README to make it descriptive

### PR DESCRIPTION
**Pull request by sireclifford**
Created html page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,9 +42,8 @@ To get a local copy up and running follow these simple example steps.
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+- GitHub: [@githubhandle](https://github.com/sireclifford)
+- Twitter: [@twitterhandle](https://twitter.com/sireclifford)
 
 👤 **Author2**
```
Comment (sireclifford) [inline README.md:48]: Remove second author since you are the only author

File: index.html [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    <div id="img-div">
+      <img src="images/image.jpg" id="image" alt="#" />
+      <p id="img-caption">Caption for image above</p>
+    </div>
+
+    <p id="tribute-info">Textual content describing person in image</p>
+    <a href="https://www.google.com" id="tribute-link" target="_blank">Read More</a>
+
+  </div>
+  <script src="js/index.js"></script>
```
Comment (sireclifford) [inline index.html:23]: This is not necessary unless your project requires it. Adding unnecessary scripts to your work can slow down your website

**Review by sireclifford — COMMENTED** [review]
Well done on practising with what you have learnt so far.
Kindly follow the comments to make your work better.

Cheers!

**Review by Kandy-Peter — CHANGES_REQUESTED** [review]
Hi @sireclifford  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Required Changes ♻️

- [ ] Please, ensure that the Pull Request you opened utilises the "Review changes" feature from GitHub instead of just competing the code. This helps users to know the PR state(Approved or requested changes). You can this short [Video](https://youtu.be/lSnbOtw4izI) for more understanding.
- [ ] I would also suggest you use summarised content as a message to the user who receives the review. You can pick an idea from mine(the look and the logic but not the content).

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by sireclifford** [comment]
@Kandy-Peter I do not understand your second suggestion.

**Review by sireclifford — COMMENTED** [review]
Hi micronaut  , 

Well done getting this far.
However there are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
- Follow Github flow ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @sireclifford   in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
(Not automatically expanded because 3 MB is too large. You can [expand it anyway](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966) or [open it in a new window](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966).)

**Comment by Kakalanp** [comment]
# STATUS: APPROVED :green_circle: 

### Your project is approved @sireclifford! :tada: 

There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :clinking_glasses: 

almost there

## Optional suggestions

- Excellent word choice [here](https://github.com/sireclifford/admission-trial-code-review/pull/1#pullrequestreview-989621273), but don't forget to [use those words everywhere](https://github.com/sireclifford/admission-trial-code-review/pull/1#discussion_r885028792)! 😉 

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**